Darktrace/darktrace_modules/client/retry.py

```diff
@@ -1,6 +1,5 @@
 import time
```

Comment:
You can also move to `urllib3` 2.x that includes `BaseHTTPResponse`. You can add an explicit dependency on that version via `poetry add urllib3@latest`

Reply:
Thank you for the advice @otetard, I will try that.